src/components/common/FooterLink/FooterLink.scss

```diff
@@ -1,16 +1,14 @@
-
+.footer {
```

Comment:
classname should be .footer-link instead of .footer. It can concide with Footer components classname